src/sentry/static/sentry/app/components/charts/chartZoom.jsx

```diff
@@ -214,10 +214,12 @@ class ChartZoom extends React.Component {
       return children(props);
     }
 
-    const dateFormat = useShortDate ? 'MMM Do' : 'lll';
+    const dateFormat = useShortDate ? 'MMM Do' : 'MMM D LT';
```

Comment:
@doralchan `MMM D LT` is equivalent to `Jun 16 8:30 PM` which should match figma mocks.